frontend/src/components/OtherImplementations.tsx

```diff
@@ -0,0 +1,79 @@
+import { useRef, useState } from "react";
+import { Overlay, Popover } from "react-bootstrap";
+import { InfoCircle } from "react-bootstrap-icons";
+import { NavLink } from "react-router-dom";
+import { Implementation } from "../data/parseReportData";
+import { mapLanguage } from "../data/mapLanguage";
+
+interface Props {
+  otherImplementationsData: Record<string, Implementation>;
+}
+
+export const OtherImplementations = ({ otherImplementationsData }: Props) => {
+  const [showPopover, setShowPopover] = useState(false);
+  const overlayRef = useRef<HTMLDivElement>(null);
+  const popoverTimeoutRef = useRef<number | undefined>(undefined);
+
+  return (
+    <div
+      ref={overlayRef}
+      className="d-flex align-items-center"
+      onMouseEnter={() => {
+        setShowPopover(true);
+        clearTimeout(popoverTimeoutRef.current);
+      }}
+      onMouseLeave={() => {
+        popoverTimeoutRef.current = window.setTimeout(() => {
+          setShowPopover(false);
+        }, 400);
+      }}
+    >
+      <div>
+        <Overlay
+          placement="left-end"
+          show={showPopover}
+          target={overlayRef.current}
+          transition={false}
+        >
+          {(props) => (
+            <Popover id="popover-basic" {...props}>
+              <Popover.Body>
+                {Object.entries(otherImplementationsData).map(
+                  ([id, impl], index) => {
+                    const implementationPath = getImplementationPath(id);
+                    return (
+                      <p key={index}>
```

Comment:
Using index as a key is generally discouraged. I think you can just use `id`? It uniquely identifies a row, right?